lib/src/main/java/edu/stanford/futuredata/macrobase/analysis/summary/apriori/ExplanationResult.java

```diff
@@ -0,0 +1,62 @@
+package edu.stanford.futuredata.macrobase.analysis.summary.apriori;
+
+
+import java.util.Map;
+import java.util.StringJoiner;
+
+public class ExplanationResult {
+    private Map<String, String> matcher;
+    private double matchedOutlier, matchedCount;
+    private double totalOutlier, totalCount;
+
+    private double rateLift;
+    private double support;
+
+    public ExplanationResult(
+            Map<String, String> matcher,
+            double matchedOutlier,
+            double matchedCount,
+            double totalOutlier,
+            double totalCount
+    ) {
+        this.matcher = matcher;
+        this.matchedOutlier = matchedOutlier;
+        this.matchedCount = matchedCount;
+        this.totalOutlier = totalOutlier;
+        this.totalCount = totalCount;
+
+        this.rateLift = (matchedOutlier / matchedCount) / (totalOutlier / totalCount);
+        this.support = matchedOutlier / totalOutlier;
+    }
+
+    public Map<String, String> getMatcher() {
+        return matcher;
+    }
+    public double matchedOutlier() {
+        return matchedOutlier;
+    }
+    public double matchedCount() {
+        return matchedCount;
+    }
+    public double rateLift() {
+        return rateLift;
+    }
+    public double support() {
+        return support;
+    }
+
+    public String prettyPrint() {
+        StringJoiner joiner = new StringJoiner(",");
+        matcher.forEach((k, v) -> joiner.add(k+"="+v));
+
+        return String.format(
+                "--\n" +
+                        "outlier rate: %d / %d\n" +
```

Comment:
This shows the rate of outliers matched / total matched for the result. I found it more useful for understanding the results than just looking at support and RR